src/VaultSharp/V1/SecretsEngines/Transit/TransitKeyCategory.cs

```diff
@@ -10,6 +10,10 @@ public enum TransitKeyCategory
 
         signing_key = 2,
 
-        hmac_key = 3
+        hmac_key = 3,
+
+        public_key = 4,
+        
+        certificate_chain = 5
```

Comment:
@Nnhaoy  can you please add cmac-key as well? and add a comment that it is enterprise only?

i'll publish a new nuget after that if you need the library with this.
if you don't need it and just did it because you noticed it missing, cool.
thanks

Reply:
@rajanadar No problem regarding the CMAC key, and don't worry about the NuGet package. I don't need it; I was just experimenting and noticed the missing part. Since I was only testing, I called the API directly via HTTP, so there's no blocker.

Reply:
Got it thanks